### PR TITLE
[FEATURE] Display Average Count

### PR DESCRIPTION
**Pull request by Pl4gue**
Resolves:  #81 

I just applied the sketch you made @jasonlong :smile: 

Screenshot:

### CODE REVIEW

File: chrome/iso.css [hunk 1]
```diff
@@ -150,6 +150,15 @@
   vertical-align: bottom;
 }
 
+.ic-stats-total-meta {
+  vertical-align: middle;
+}
+
+.ic-stats-average {
+  font-weight: bold;
+  color: black;
```
Comment (jasonlong) [inline chrome/iso.css]: Let's kill the `color` rule here and have the default GitHub `body` color come through (`#24292e`). 

Related: [this is a great tip](https://ianstormtaylor.com/design-tip-never-use-black/).

Comment (Pl4gue) [reply chrome/iso.css]: Hey @jasonlong , thanks a lot for sharing this article, it was really interesting to read :slightly_smiling_face:

**Comment by Pl4gue** [comment]
Just applied the change, but I cannot kill the color rule as it would inherit the #777 from `.ic-stats-label`

**Comment by jasonlong** [comment]
Ahh good point. This looks good to me. I'll get a new release built after merging shortly. :thumbsup:

Thanks for the contribution. 😸

**Pull request merged**